CMakeLists.txt

```diff
@@ -9,15 +9,17 @@
 
 cmake_minimum_required( VERSION 3.12 )
 
-find_package( ecbuild 3 REQUIRED HINTS ${CMAKE_CURENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../ecbuild )
 
 project( iodaconv VERSION 0.0.1 LANGUAGES CXX Fortran )
+list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
 
+find_package( ecbuild 3 REQUIRED HINTS ${CMAKE_CURENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../ecbuild )
```

Comment:
```suggestion
find_package( ecbuild 3 REQUIRED )
```

Reply:
Done in https://github.com/JCSDA-internal/ioda-converters/pull/1566/commits/89ff8650c21b508f57ec5a42463d3778cb05eca3.